dask/distributed/__init__.py

```diff
@@ -2,3 +2,10 @@
 from .scheduler import Scheduler
 from .client import Client
 from .ipython_utils import dask_client_from_ipclient
+
+from warnings import warn
+
+warn(FutureWarning('''
```

Comment:
Should probably be a `DeprecationWarning` instead. `FutureWarning` is about future semantic changes, not deprecated modules.